<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtAdminUnverifiedCreditsUI</class>
 <widget class="QWidget" name="QtAdminUnverifiedCreditsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtAdminUnverifiedCreditsUI</string>
  </property>
  <widget class="QPushButton" name="aprovePushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>260</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>APROVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getInfoPushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>GET INFO</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
